tests/kubeutils.py

```diff
@@ -0,0 +1,117 @@
+import time
+
+from kubernetes import client
+from kubernetes import watch
+from kubernetes.client import rest
+from kubernetes.stream import stream
+
+
+RESOURCE_READY = 0
+RESOURCE_EMPTY = 1
+RESOURCE_MISSING = 2
+
+
+def assert_resource_availability(resource, node, status, timeout=0):
+    v1 = client.CoreV1Api()
```

Comment:
Can it be a module/session fixture or just a global?

Reply:
I would need to pass the fixture everywhere, to all helper functions. I don't want to do that.

I will try to make it global, but I think it broke it the last time.